packages/editor/src/components/parts/common/responsible/RoleSelect.tsx

```diff
@@ -13,15 +17,28 @@ type RoleSelectProps = {
 };
 
 const RoleSelect = ({ value, onChange, inputProps, showtaskRoles }: RoleSelectProps) => {
-  const roleItems = useRoles(showtaskRoles);
+  const { roles: roleItems } = useRoles(showtaskRoles);
+  const browser = useBrowser();
+  const path = usePath();
   const selectedRole = useMemo<SelectItem | undefined>(() => {
     if (value) {
       return roleItems.find(e => e.value === value) ?? { label: value, value };
     }
     return DEFAULT_ROLE;
   }, [value, roleItems]);
 
-  return <Select items={roleItems} value={selectedRole} onChange={item => onChange(item.value)} inputProps={inputProps} />;
+  return (
+    <div className='role-select'>
+      <Select items={roleItems} value={selectedRole} onChange={item => onChange(item.value)} inputProps={inputProps} />
+      <Browser
+        {...browser}
+        types={['role']}
+        location={path}
```

Comment:
also something which is maybe implemented like this since a time, but why do you need to set the location as a path? Can't you just get the path context from within the Browser component?
Or is this location not always the path? Then maybe the different browser implementations can get this path context if they need it and the others use an option?

Reply:
That's quite possible. I'll also analyze this in a later step and if possible make improvements .